pkg/controller/tuned/tuned_controller.go

```diff
@@ -249,6 +291,25 @@ func (r *ReconcileTuned) syncDaemonSet(tuned *tunedv1.Tuned) error {
 		}
 	} else {
 		glog.V(2).Infof("Tuned DaemonSet already exists, updating")
+
+		operatorReleaseVersion := os.Getenv("RELEASE_VERSION")
+		operandReleaseVersion := ""
+
+		for i, e := range daemonset.Spec.Template.Spec.Containers[0].Env {
```

Comment:
Given this new snippet appears twice, may be worth putting in it's own function?

Reply:
Also not necessarily an issue, I think using both the key and value is unnecessary.